Mert Musluoğlu Ödevi pullRequest

musluogluu opened the pull request:
Mert Musluoğlu'nun ödevi pull request @lingaspeakingspider @cpyjavascript

musluogluu committed:
Add files via upload

thunderstorm010 commented:
Kod çalışmamakta @musluogluu

musluogluu commented:
Nasıl yani ben açtığımda çıktı alıyordum çalışmamaktan kasıt ne


29 Eyl 2024 Paz, saat 20:13 tarihinde thunderstorm010 <
***@***.***> şunu yazdı:

> Kod çalışmamakta @musluogluu <https://github.com/musluogluu>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cpyjavascript/CALROV-2025-Egitim/pull/9#issuecomment-2381431013>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLRWBDHJ6ZIUNYDDPTRGO4LZZAYNBAVCNFSM6AAAAABPBZRX46VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGOBRGQZTCMBRGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

thunderstorm010 commented on MertMusluogluOdev.py:
```diff
@@ -0,0 +1,49 @@
+        # Mert, çıktında olmadık yerde "metre" yazısı varsa yukardakini sil boş boş bakma koda.
+
+    def heartbYazdır(self):
+        if hasattr(self.heartbYazdır, 'HEARTBEAT'):
```
??????? Fonksiyonun attribute'ı olmaz ki

musluogluu replied:
aaa harbiden bozukluk varmış hemen bakıyorum şimdi

thunderstorm010 commented on MertMusluogluOdev.py:
```diff
@@ -0,0 +1,49 @@
+    irtifa_thread = Thread(target= actionObjects.irtifaYazdır)
+    heartbeat_thread = Thread(target= actionObjects.heartbYazdır)
+
+    irtifa_thread.start()
```
Burada sıkıntı yok, fakat mantıksal olarak threading yapmanın bir anlamı yok çünkü uzun süren v.s. işlem yapılmıyor fonksiyonlarda. Oğuz/Kaan'ın burda istediği muhtemelen iki ayrı threadde aynı anda recv_match'i type argümanıyla kullanıp bu mesajları iki ayrı threadde almanız. Tabi burada sıkıntı yok ama mantıksal olarak bi tık hatalı

musluogluu replied:
yo bende hak veriyorum mantıksal hata var dalgınlık olmuş tabi sorunsuz çalışınca da fark etmedim

musluogluu replied:
Kodu değiştirdim ve şu anda gördüğüm kadarıyla bi sıkıntı yok istersen bakabilirsin. @thunderstorm010

musluogluu replied:
birde sen burak y. misin

musluogluu committed:
New MertMusluogluOdev.py

musluogluu committed:
Update2 MertMusluogluOdev.py

ABİLER ASIL ÖDEV BU BAZI DEĞİŞİKLİKLER OLDU

musluogluu committed:
deger değişimi MertMusluogluOdev.py

asıl ödev bu sadece irtifa -> attidute yapıldı kötü ingilizcemden dolayı altidute zannetim

musluogluu committed:
NORMAL MertMusluogluOdev.py
